blockchain/state_migration.go

```diff
@@ -494,14 +512,19 @@ func (bc *BlockChain) StartWarmUp() error {
 		return err
 	}
 	// run goroutine for each child node
-	resultCh := make(chan struct{}, 10000)
+	resultCh := make(chan int, len(children))
 	errCh := make(chan error)
 	bc.quitWarmUp = make(chan struct{})
 	for _, child := range children {
-		go bc.iterateStateTrie(child, db, resultCh, errCh)
+		stateDB, err := state.New(child.Unextend(), db, nil, nil)
+		if err != nil {
+			return err
```

Comment:
Unless you're trying to change the error handling, I think it should be:

```suggestion
			continue
```

Reply:
Thank you. Updated.